loglist2/loglist2_test.go

```diff
@@ -20,15 +20,26 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"log"
 	"reflect"
 	"sort"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/kylelemons/godebug/pretty"
+	//"github.com/kylelemons/godebug/pretty"
+	"github.com/sergi/go-diff/diffmatchpatch"
 )
 
+// parseValidUnixTime is helper func
+func parseValidUnixTime(sTime string) time.Time {
```

Comment:
nit: Typically, functions like this start with "must", e.g. "mustParseUnixTime".

Reply:
Renamed